README.md

```diff
@@ -203,6 +203,22 @@ yarn run test-gui-openshift
 This will include the normal k8s CRUD tests and CRUD tests for OpenShift
 resources.
 
+#### Using specific version of Chrome browser
+
+Integration tests are run in a headless Chrome driven by a [ChromeDriver](https://chromedriver.chromium.org/downloads). Each ChromeDriver
+supports specific Chrome versions.
+
+By default test use the Chrome browser installed by the system, it is also possibly to [download](https://www.chromium.org/getting-involved/download-chromium) a specific version
+of Chrome browser by setting a [branch position](https://omahaproxy.appspot.com/) and sha256sum (of zip package) using environment variables.
+Downloading chrome requires `curl`, `unzip` and `sha256sum` command line utilities installed.
```

Comment:
nit: oxford comma please :)

```suggestion
Downloading chrome requires `curl`, `unzip`, and `sha256sum` command line utilities installed.
```

Reply:
You should mention this only works on Linux.

Reply:
now I know what "oxford comma" is :-) , done, the linux thing too !